tests/validators/test_definitions.py

```diff
@@ -46,7 +46,7 @@ def test_check_ref_used_ignores_metadata():
         )
     )
     assert v.validate_python([1, 2, 3]) == [1, 2, 3]
-    assert plain_repr(v).endswith('slots=[])')
+    # assert plain_repr(v).endswith('slots=[])')
```

Comment:
If we accept that we'll be using slots here I want to either (1) replace the repr or (2) remove it from the test. let me know what you think @samuelcolvin